code/game/objects/structures/crates_lockers/closets.dm

```diff
@@ -120,6 +124,9 @@
 	if(access_choices)
 		access_choices = card_reader_choices
 
+	if(is_station_level(z))
```

Comment:
I don't feel the roundstart filtering is needed, since this list is only used at roundstart anyways.

I sort of just assumed that crates weren't closet subtypes and would not be included. I've been doing my testing on Runtimestation, which doesn't have any crates on it, so I never really got any indicators that things weren't working as intended (OOPS).

I want this to use lockers specifically since they're more prevalent in people's job spaces. More liable to be stumbled across very early into the round, rather than sitting as an extra piece of maint loot for an assistant to hoard.

Reply:
I've elected to make it an actual roundstart list (I think my intent was to do this I just forgot to put the mapload there) and the list now properly filters crates.